src/Commands/Site/InfoCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
This is stolen from https://github.com/pantheon-systems/terminus/pull/1211, I'll rebase on that when it gets merged.